change_of_vars_demo1d/change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    try:
</pre><p>Can you check the new instructions on the README page and modify the code accordingly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>